docs/run/_toc.json

```diff
@@ -96,6 +96,10 @@
         {
           "title": "Retired systems",
           "url": "/run/retired-systems"
+        },
+        {
+          "title": "Hardware considerations and limitations for classical feedforward and control flow",
```

Comment:
We currently truncate page titles in the left sidebar. The frontend team is working on that. But in the meantime, any ideas for a shorter name?

Reply:
Not really. Since it's not a problem in principle and the issue will be fixed, I would like to leave it as is.